docs/cli/shell.rst

```diff
@@ -1,9 +1,10 @@
 lxdock shell
 ============
 
-**Command:** ``lxdock shell [arguments] [name]``
+**Command:** ``lxdock shell [arguments] [name] [-c command_line]``
```

Comment:
I think that ``lxdock shell [name] [arguments]`` would be more appropriate. Command arguments are not explicitly defined in this line in other command reference pages, so It'd be best to try to remain consistent here too. 😉